source/plugins/base/queries/user.x.graphql

```diff
@@ -30,7 +30,7 @@ query BaseUserX {
     repositoriesContributedTo(includeUserRepositories: true) {
       totalCount
     }
-    repositories(last: 0) {
+    repositories(last: 0, ownerAffiliations : OWNER) {
```

Comment:
I think this should use `$affiliations` to use the constraint set by [`repositories_affiliations`](https://github.com/lowlighter/metrics/tree/master/source/plugins/base#repositories_affiliations)

Reply:
Actually When I used `$affliations` then I was able to retrieve only my repositories but not other users. So using `$affliations` will create issue in metrics(web) but `$ownerAffiliations` is able to give us the repositories count of every user.

Reply:
done😁